src/web/lib/decidePalette.ts

```diff
@@ -347,11 +347,13 @@ const fillCaptionTriangle = (format: Format): string => {
 };
 
 const fillCardIcon = (format: Format): string => {
+	// Setting Card clock colour for immersive cards to all be dark grey
+	if (format.display === Display.Immersive) return neutral[60];
```

Comment:
I think this is a merge conflict gone awry? These lines aren't on `main` and don't seem to be related

Reply:
I think these should be there from this PR rather than  `main`. There's the pictures stuff in here as well as a few small design fixes. PR for this should be here: https://github.com/guardian/dotcom-rendering/pull/2539/commits/800012b14819b27f71e18630ff8f3da46396de0c